src/pages/workspace/WorkspaceProfilePage.tsx

```diff
@@ -116,11 +139,6 @@ function WorkspaceProfilePage({policy, currencyList = {}, route}: WorkSpaceProfi
         }
     }, [policy?.id, policyName, activeWorkspaceID, setActiveWorkspaceID]);
 
-    // When we create a new workspaces, the policy prop will not be set on the first render. Therefore, we have to delay rendering until it has been set in Onyx.
-    if (policy === undefined) {
-        return null;
-    }
-
```

Comment:
Nice one!
This fixes issue where we have a little delay after creating workspace 👍